components/campaigns/AnnoucementHero.js

```diff
@@ -25,7 +25,7 @@ export default function AnnouncementHero({ className = '', small = false, hideVi
   const [activeIndex, setActiveIndex] = useState(0);
 
   const cfpDeadlineIndia = '2023-11-30T06:00:00Z'
-  const cfpDeadlineFrance = '2023-12-6T06:00:00Z'
+  const cfpDeadlineFrance = '2023-12-06T06:00:00Z'
```

Comment:
@derberg We should value a `0` as well, in order to not break the website text in such a way 😅